docs/examples/resources.rst

```diff
@@ -183,6 +183,22 @@ Delete multiple records at once by filters using filter format from search.
         ('Field_2', 'equals', value2)
     )
 
+Batch Record Modify
+^^^^^^^^^^^^^^^^^^^
+
+Bulk modify records by filter or by list of records. Invalid function inputs will cause entire operation to fail.
+
+.. code-block:: python
+
+    # List of all records in app
+    records = app.records.search(limit=0)
```

Comment:
Grab a couple records explicitly like in the other doc, and split filters vs records into two separate code-blocks